src/gxhash/mod.rs

```diff
@@ -116,15 +115,19 @@ pub(crate) unsafe fn compress_all(input: &[u8]) -> State {
 }
 
 #[inline(always)]
-unsafe fn compress_many(mut ptr: *const State, hash_vector: State, remaining_bytes: usize) -> State {
-
+unsafe fn compress_many(
+    mut ptr: *const State,
+    hash_vector: State,
+    remaining_bytes: usize,
+) -> State {
     const UNROLL_FACTOR: usize = 8;
 
-    let unrollable_blocks_count: usize = remaining_bytes / (VECTOR_SIZE * UNROLL_FACTOR) * UNROLL_FACTOR; 
+    let unrollable_blocks_count: usize =
+        remaining_bytes / (VECTOR_SIZE * UNROLL_FACTOR) * UNROLL_FACTOR;
     let end_address = ptr.add(unrollable_blocks_count) as usize;
     let mut hash_vector = hash_vector;
-    while (ptr as usize) < end_address {
 
+    while (ptr as usize) < end_address {
```

Comment:
I like to keep variables relative to a loop next to the loop, this is why there is no empty line above this while loop